docs/project-administration/releasing/mathics-core.rst

```diff
@@ -0,0 +1,142 @@
+==============================
+Making a Mathics3 Core Release
+==============================
+
+.. code::
+
+    $ PACKAGE="Mathics3"
+    $ package="mathics-core"
+    # Edit mathics/version.py
+    $ source mathics/version.py # to set in POSIX shell
+    $ echo $__version__
+
+Workflows update?
+=================
+
+Check ``.github/workflows/*.yml`` to make see if we are using
+github versions (of scanner) for testing. If so adjust.
+
+
+Create a new branch
+===================
+
+.. code::
+
+    $ git checkout -b release-$__version__
+    $ git commit -m"Get ready for release $__version__" .
+
+Update Changes
+==============
+
+.. code::
+
+    $ make ChangeLog
+
+Update ``CHANGES.rst`` from ``ChangeLog``
+
+::
+
+    $ make check
+    $ git commit --amend .
+    $ git push -u origin HEAD # get CI testing going early
+
+Make Python PDF
+======================
+
+.. code::
+
+   $ pyenv local 3.11 # or whatever latest Python you want ot build doc is
+   $ make clean # make sure mathics-title.pdf has not been removed
+   $ make doc
+
+
+Look at resulting PDF and colophon
+
+Check Python versions
+======================
+
+.. code::
+
+   $ ./admin-tools/check-versions.sh
+
+Merge release
+=============
+
+Go to github and merge release into master.
+
+Then...
+.. code::
+::
+
+    $ git checkout master
+    $ git pull
+
+
+Check package from github
+=========================
+
+TODO: turn this into a script in ``admin-tools``
+
+.. code::
+
+    $ [[ ! -d /tmp/gittest ]] && mkdir /tmp/gittest; pushd /tmp/gittest
+    $ pyenv local pyston-2.3.5 # Use a version that is not the most recent
+    $ pip install -e git+https://github.com/Mathics3/${package}.git#egg=${PACKAGE}
```

Comment:
here ${PACKAGE} should be `mathics-core`, isn't it?

Reply:
> here ${PACKAGE} should be `mathics-core`, isn't it?

Yes, that is correct it is set on line 7. The variables `package` is probably a poor choice. Probably, `github-top-dir` would be better.  I tend to adjust the notes as I go do the release, so there is a bit of sloppiness for a while.